src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -165,7 +169,9 @@ object Parameters {
     PARAM_USE_COPY_UNLOAD -> "true",
     PARAM_USE_PROXY -> "false",
     PARAM_EXPECTED_PARTITION_COUNT -> "1000",
-    PARAM_MAX_RETRY_COUNT -> "10"
+    PARAM_MAX_RETRY_COUNT -> "10",
+    // 2020-02-13 Default authenticator
+    PARAM_AUTHENTICATOR -> "snowflake"
```

Comment:
Is "snowflake" a valid authenticator?

Reply:
yes, per the Snowflake JDBC doc, snowflake is the default authenticator (i.e. user/password) https://docs.snowflake.net/manuals/user-guide/jdbc-configure.html#connection-parameters